ruby/lib/miscreant/aead.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: binary
+# frozen_string_literal: true
+
+module Miscreant
+  # The AEAD class provides Authenticated Encryption with Associated Data
+  #
+  # If you're looking for the API to encrypt something, congratulations!
+  # This is the one you probably want to use. This class provides a high-level
+  # interface to Miscreant's misuse-resistant encryption.
+  class AEAD
+    # Create a new AEAD encryptor instance.
+    #
+    # You will need to select an algorithm to use, passed as a string:
+    #
+    # * "AES-SIV" (RFC 5297): the original AES-SIV function, based on CMAC
+    # * "AES-PMAC-SIV": a parallelizable AES-SIV alternative
+    #
+    # Choose AES-PMAC-SIV if you'd like better performance.
+    # Choose AES-SIV if you'd like wider compatibility: AES-PMAC-SIV is
```

Comment:
Should this line be `AES-CMAC-SIV` rather than `AES-PMAC-SIV`?

Reply:
I think the description is just bad... what it's trying to say is that AES-PMAC-SIV is only available in the Miscreant libraries, so if you want a more standard construction, go with AES-SIV.